qiskit/visualization/pass_manager_visualization.py

```diff
@@ -59,10 +59,18 @@ def pass_manager_drawer(pass_manager, filename=None, style=None, raw=False):
         ImportError: when nxpd or pydot not installed.
```

Comment:
I'm not sure about these changes, since they are not related to the Pauli feature map. Could you comment on them or move them to a different new PR (then I can accept this one)?